### PR TITLE
moyo

### PR DESCRIPTION
**Pull request by ZoyaCanepa**
- [DEMO LINK](https://baranovskazoya.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://baranovskazoya.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,65 @@
...
+          <li class="nav__item">
+            <a href="#samsung"
+                class="nav__link">
+                Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones"
+                 class="nav__link">
+                 Smartphones
```
Comment (OleksiiBlezniuk) [inline src/index.html:33]: Please fix indentations everywhere

Comment (OleksiiBlezniuk) [inline src/index.html:33]:
```suggestion
            <a
              href="#smartphones"
              class="nav__link"
            >
              Smartphones
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,65 @@
...
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  display: inline-block;
+  line-height: 60px;
+  transition: color 0.3s;
+}
+
+.nav__link:hover,
+.is-active {
```
Comment (OleksiiBlezniuk) [inline src/style.css:48]:
```suggestion
.is-active:hover {
```

Comment (OleksiiBlezniuk) [reply src/style.css:48]: No need to increase selector specifity

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Good job! There are few comments to fix

File: src/style.css (hunk 2 above)
Comment (baranovskazoya) [reply src/style.css:48]: З цим виправленням у мене не проходить тоді 4 тести:
file:///C:/projects/layout_moyo-header/backstop_data/html_report/index.html

Screenshot_1
Або я не зрозуміла, як правильно зробити

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,67 @@
...
+      <div class="logo">
+        <a href="#home"><img src="images/logo.png"alt="moyo-logo"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+            class="nav__link is-active">
+            Apple
+            </a>
```
Comment (OleksiiBlezniuk) [inline src/index.html:22]:
```suggestion
            <a href="#apple" class="nav__link is-active">
              Apple
            </a>
```

Comment (OleksiiBlezniuk) [reply src/index.html:22]: If there are 2 or 1 attributes, you can leave them at one line (last time I showed you wrong example, sorry).
If there are 3 or more - every attributes should starts from new line

Nested elements (or text content) should have indentation 2 spaces from its parent indentation

Comment (OleksiiBlezniuk) [reply src/index.html:22]: Fix it everywhere, it will make your code cleaner

File: src/index.html [hunk 4]
```diff
@@ -6,9 +6,67 @@
...
+            class="nav__link">
+            Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops&computers"
+            class="nav__link"
+            data-qa="hover">
+            Laptops & Computers
+            </a>
```
Comment (OleksiiBlezniuk) [inline src/index.html:41]:
```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Comment (OleksiiBlezniuk) [reply src/index.html:41]: And here is the case when there are more then 2 attributes, so every attribute should start from new line.

Also pay attention that closing bracket has same indentation as open

Please fix everywhere

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Please pay attention to code style

**Review by OleksiiBlezniuk — APPROVED** [review]
LGTM! 👍

**Pull request closed without merging**